qutebrowser/config/config.py

```diff
@@ -27,6 +27,7 @@
 
 from PyQt5.QtCore import pyqtSignal, QObject, QUrl
 
+from qutebrowser.commands import cmdexc, runners
```

Comment:
The `runners` import here is what causes the mypy failures :thinking: 

Don't know why exactly, I guess something changes about the (circular?) import handling in mypy...

This seems to help:

```diff
diff --git i/qutebrowser/keyinput/modeparsers.py w/qutebrowser/keyinput/modeparsers.py
index 48f3594a5..fe03a9db6 100644
--- i/qutebrowser/keyinput/modeparsers.py
+++ w/qutebrowser/keyinput/modeparsers.py
@@ -86,6 +86,8 @@ class NormalKeyParser(CommandKeyParser):
         _partial_timer: Timer to clear partial keypresses.
     """
 
+    _sequence: keyutils.KeySequence
+
     def __init__(self, *, win_id: int,
                  commandrunner: 'runners.CommandRunner',
                  parent: QObject = None) -> None:
@@ -154,6 +156,8 @@ class HintKeyParser(basekeyparser.BaseKeyParser):
         _last_press: The nature of the last keypress, a LastPress member.
     """
 
+    _sequence: keyutils.KeySequence
+
     def __init__(self, *, win_id: int,
                  commandrunner: 'runners.CommandRunner',
                  hintmanager: hints.HintManager,

```

Could you apply that to your branch to fix the mypy failure please?